proxy/http/ForwardedConfig.cc

```diff
@@ -0,0 +1,126 @@
+/** @file
+
+  Configuration of Forwarded HTTP header option.
+
+  @section license License
+
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+
+#include <bitset>
+#include <string>
+#include <cctype>
+
+#include <ts/string_view.h>
+#include <ts/MemView.h>
+
+#include <HttpConfig.h>
+
+namespace HttpForwarded
+{
+OptionBitSet
+optStrToBitset(ts::string_view optConfigStr, std::string &error)
+{
+  using SV = ts::StringView;
```

Comment:
I'm not sure I'm keen on these "convenience" abbreviations. Why not use ts;:StringView and OptionBitset constantly? It makes for easier to read code, since I don't have to go look up what the abbreviated (SV and OBS) symbols are (and neither are obvious).

Reply:
I think you meant to write:

Zwoop is not sure Zwoop is keen on these "convenience" abbreviations. Why not use ts;:StringView and OptionBitset constantly? Using ts::StringView and OptionBitset constantly makes for easier to read code, since zwoop does not have to go look up what the abbreviated (SV and OBS) symbols are (and neither are obvious).

Reply:
Walt, this is one reason I use a desktop machine for actual development. We can get yours setup and put a couple of 24" monitors on it to handle longer variable / class names. Overall I side with zwoop on this, it's not a good habit to in effect introduce your own mini-language for code others might have to maintain.